streaming/src/main/java/org/apache/carbondata/streaming/CarbonStreamRecordWriter.java

```diff
@@ -188,7 +190,7 @@ private void initializeAtFirstRow() throws IOException {
       compressorName = carbonTable.getTableInfo().getFactTable().getTableProperties().get(
           CarbonCommonConstants.COMPRESSOR);
       if (null == compressorName) {
-        compressorName = CompressorFactory.getInstance().getCompressor().getName();
+        compressorName = CompressorFactory.NativeSupportedCompressor.SNAPPY.getName();
```

Comment:
why always snappy for streaming segment ? before I think it was supporting configured compressor

Reply:
There is testcase failing if set to ZSTD. Need another PR for it

Reply:
ok, please add a TODO